includes/admin/upgrades/upgrade-functions.php

```diff
@@ -1425,14 +1425,14 @@ function edd_upgrade_render_v30_migration() {
 
 	<div id="edd-migration-ready" class="notice notice-success" style="display: none;">
 		<p>
-			<?php _e( '<strong>Database Upgrade Complete:</strong> All database upgrades have been completed.', 'easy-digital-downloads' ); ?>
+			<?php echo wp_kses( __( '<strong>Database Upgrade Complete:</strong> All database upgrades have been completed.', 'easy-digital-downloads' ), array( 'strong' => true ) ); ?>
```

Comment:
I believe the format is supposed to be:

```suggestion
			<?php echo wp_kses( __( '<strong>Database Upgrade Complete:</strong> All database upgrades have been completed.', 'easy-digital-downloads' ), array( 'strong' => array() ) ); ?>
```

(It might work both ways but I most often see it as `array()` :shrug: )

Reply:
@ashleyfae Both do work. I've switched to using `true` for items that don't have any additional attributes lately.

Reply:
I guess that's fine then. WordPress core seems to use `array()` for the top level ones, but `true` does make more sense to me.

Reply:
Although it does flag as an error in PhpStorm with `true`, which is irksome.

![Screenshot from 2020-05-14 15-38-09](https://user-images.githubusercontent.com/6324272/81947863-f4d13200-95f8-11ea-88a0-8fe57ac866f8.png)

This is due to the docs for `wp_kses()` being `array[]`

```
@param array[]|string $allowed_html
```

Reply:
@ashleyfae There is a mix in core for top level items that don't have further attributes: https://github.com/WordPress/WordPress/blob/001ffe81fbec4438a9f594f330e18103d21fbcd7/wp-admin/includes/class-automatic-upgrader-skin.php#L86-L96

We can stick with `array()` for all if that helps IDEs though.